POP.nix

```diff
@@ -222,7 +222,19 @@
     computePrecedenceList = c3ComputePrecedenceList;
     mergeInstance = mergeAttrset;
     bottomInstance = {};
-    topProto = __meta__: self: super: super // {inherit __meta__;};
+    topProto = __meta__: self: super:
+      super
+      // {
+        __meta__ =
+          __meta__
```

Comment:
Are you sure that the `__meta__` here isn't circularly defined, vs referring to the previous binding?

Reply:
The previous line was just `inherit __meta__`, so this should have the same effect, but with some additions